src/client/api/client_internal.h

```diff
@@ -56,6 +56,8 @@ struct daos_event_private {
 	unsigned int		is_barrier:1;
 	/** flag to indicate whether to convert DER to errno */
 	unsigned int		is_errno:1;
+	/** flag to indicate whether even is thread private */
```

Comment:
s/even/event

Reply:
ok if i get +1s and don't need to re-push, i will change it with skip-test pragma (but will need force landing). queue is about 30hrs for testing.